src/main/kotlin/skill/GuiZha.kt

```diff
@@ -83,8 +84,9 @@ class GuiZha : MainPhaseSkill() {
             player.getSkillUseCount(SkillId.GUI_ZHA) == 0 || return false
             val game = player.game!!
 
-            // 当前场上只要有一个队友（包括自己）没听牌就利诱
-            if (!game.players.any { it!!.alive && it.isPartnerOrSelf(player) && it.messageCards.count(it.identity) == 2 }) {
+            // 当前场上只要有一个队友（包括自己）没听牌，或自己是搅局者，就利诱
+            if (player.identity === Black && player.secretTask == Disturber || player.identity !== Black &&
```

Comment:
```suggestion
            if (player.identity == Black && player.secretTask == Disturber || player.identity != Black &&
```
`===`仅用于比较是否为同一个对象，你这个是值，只需要`==`就行了。